plugins/idea/pluginImpl/src/main/java/com/imaginea/kodebeagle/impl/ui/MainWindow.java

```diff
@@ -48,13 +46,17 @@ public static boolean scalaPluginInstalledAndEnabled() {
 
     @Override
     protected final boolean checkOptionalDependency() {
+        /*
+        TODO to be implememnted for scala support
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)